### PR TITLE
feat(tracking): Record the page referrer for every event.

### PR DESCRIPTION
**Pull request by judahtanthony**
### Overview

Record the page referrer for every event.

### PR Checklist

- [x] Related to JIRA ticket: REACH-1027
- [ ] I have run this code to verify it works
- [ ] This PR includes unit tests for the code change

### CODE REVIEW

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/tracking@0.8.1-alpha.67fdf7.0<br/></pre>

File: packages/tracking/src/events/track.ts [hunk 1]
```diff
@@ -39,6 +39,7 @@ export const createTracker = ({ apiBaseUrl, verbose }: TrackerOptions) => {
       path: window.location.pathname,
       title: window.document.title,
       url: window.location.href,
+      referer: window.document.referrer,
```
Comment (MattSchiller) [inline packages/tracking/src/events/track.ts:42]: nit: `referrer` with 2 `r`

**Review by MattSchiller — APPROVED** [review]
Happy to thumb this one, but there's a typo that should get fixed first 👍

File: packages/tracking/src/events/track.ts (hunk 1 above)
Comment (judahtanthony) [reply packages/tracking/src/events/track.ts:42]: [Proof](https://en.wikipedia.org/wiki/HTTP_referer) that all engineers are poor spellers.

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/tracking@0.8.1-alpha.03cbfc.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://60c1271553825400c1d013f7--gamut-preview.netlify.app>https://60c1271553825400c1d013f7--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/60c1271553825400c1d013f7>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/tracking@0.8.1-alpha.407849.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://60c14ad9f551f13bb17b6642--gamut-preview.netlify.app>https://60c14ad9f551f13bb17b6642--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/60c14ad9f551f13bb17b6642>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/tracking@0.8.1-alpha.0c8006.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://60c16e9053825463e9d013b2--gamut-preview.netlify.app>https://60c16e9053825463e9d013b2--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/60c16e9053825463e9d013b2>Deploy Logs</a></p>

**Pull request merged**